autonomy/deploy/generators/docker_compose/base.py

```diff
@@ -132,15 +132,17 @@ def generate_config_tendermint(self) -> "DockerComposeGenerator":
 
         return self
 
-    def generate(
-        self,
-    ) -> "DockerComposeGenerator":
+    def generate(self, image_version: Optional[str] = None) -> "DockerComposeGenerator":
         """Generate the new configuration."""
 
+        image_version = image_version or self.service_spec.service.agent.hash
```

Comment:
The whole hash? I think it might be better to take the trailing 6 characters say

Reply:
Probably a stupid question, but is there a chance of the last 6 characters of an hash for a package being same?

Reply:
Yes there is a chance. The hash is fine, just quite long. In the interest of getting to simple and done let's leave as is